lms/templates/learner_dashboard/program_details_2017.html

```diff
@@ -15,6 +15,8 @@
 <%static:require_module module_name="js/learner_dashboard/program_details_factory_2017" class_name="ProgramDetailsFactory2017">
 ProgramDetailsFactory2017({
     programData: ${program_data | n, dump_js_escaped_json},
+    courseData: ${course_data | n, dump_js_escaped_json},
+    certificateData: ${certificate_data | n, dump_js_escaped_json},
```

Comment:
@MatthewPiatetsky I went ahead and added `courseData` to the template here. I'm using that name to keep it consistent with `programData` and `certificateData`. You should be able to pull the data you need for the detail page from this.